website/docs/docs/cloud/migration.md

```diff
@@ -15,6 +15,13 @@ Your account will be automatically migrated on or after its scheduled date. Howe
 
 ## Recommended actions
 
+:::info Rescheduling your migration
+
+Customers on dbt Cloud Enterprise tiers can postpone their account migration by up to 45 days. To reschedule your migration, go to **Account Settings** → **Migration guide**.
+
+For help, contact us at [sales@getdbt.com](mailto:sales@getdbt.com).
```

Comment:
Should this be our support email instead, or is sales appropriate for Enterprise accounts?

Reply:
@aaronraff I believe this should be `support@getdbt.com`.